packages/common/src/Utility.ts

```diff
@@ -268,9 +268,20 @@ SimpleSelectionMixin.prototype.deserializeState = function (state) {
+export function faKeys(): string[] {
+    return Object.keys(faCharMap);
+}
+
+export function faChar(key: string, defChar: string = "fa-question"): string {
+    if (key.indexOf("fa-") === 0) {
+        return faCharMap[key] || faCharMap[defChar];
+    }
+    return key;
```

Comment:
If the key doesn't start with "fa-" maybe we should throw an error?

Reply:
No it supports both a single char (copy / paste from the cheat sheet) or the fa-* name.